openupgrade_scripts/scripts/product/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,91 @@
+---Models in module 'product'---
+obsolete model report.product.report_producttemplatelabel [abstract]
+new model product.catalog.mixin [abstract]
+new model product.document
+new model report.product.report_producttemplatelabel2x7 [abstract]
+new model report.product.report_producttemplatelabel4x12 [abstract]
+new model report.product.report_producttemplatelabel4x12noprice [abstract]
+new model report.product.report_producttemplatelabel4x7 [abstract]
+# NOTHING TO DO
+
+---Fields in module 'product'---
+product      / product.attribute        / display_type (selection)      : selection_keys is now '['color', 'multi', 'pills', 'radio', 'select']' ('['color', 'pills', 'radio', 'select']')
+# NOTHING TO DO: add new selection: multi
+
+product      / product.attribute.value  / default_extra_price (float)   : NEW
+product      / product.attribute.value  / image (binary)                : NEW attachment: True
+product      / product.category         / product_properties_definition (properties_definition): NEW
+# NOTHING TO DO: new feature
+
+product      / product.document         / _inherits                     : NEW _inherits: {'ir.attachment': 'ir_attachment_id'}
+product      / product.document         / active (boolean)              : NEW hasdefault: default
+product      / product.document         / ir_attachment_id (many2one)   : NEW relation: ir.attachment, required
+# DONE: populated in post-migration with attachments on products
+
+product      / product.packaging        / product_id (many2one)         : now required
+# NOTHING TO DO
+
+product      / product.pricelist        / _order                        : _order is now 'sequence asc, id asc' ('sequence asc, id desc')
+product      / product.pricelist        / activity_ids (one2many)       : NEW relation: mail.activity
+product      / product.pricelist        / message_follower_ids (one2many): NEW relation: mail.followers
+product      / product.pricelist        / message_ids (one2many)        : NEW relation: mail.message
+# NOTHING TO DO
+
+product      / product.pricelist.item   / active (boolean)              : DEL
+# NOTHING TO DO
+
+product      / product.product          / activity_user_id (many2one)   : not related anymore
+product      / product.product          / activity_user_id (many2one)   : now a function
+product      / product.product          / message_main_attachment_id (many2one): DEL relation: ir.attachment
+product      / product.product          / product_document_ids (one2many): NEW relation: product.document
```

Comment:
You should probably mark these 2 lines as DONE in post-migration instead of NOTHING TO DO

Reply:
for the first one see my [question](https://github.com/OCA/OpenUpgrade/pull/4600#issuecomment-2427155081) in the mail PR, agreed with the second, done in 074f09d36537 and will squash when you agree